src/styles/header/header_style.scss

```diff
@@ -0,0 +1,62 @@
+.page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
+  margin: 0;
+}
```

Comment:
margin would be better to reset in reset file, and font you can set as default in main.scss or create normalize file